codersdk/deployment.go

```diff
@@ -1497,6 +1498,16 @@ Write out the current server config as YAML to stdout.`,
 			Default:     "", // empty string means pick best server
 			Hidden:      true,
 		},
+		{
+			Name:        "Proxy Health Check Interval",
+			Description: "The interval in which coderd should be checking the status of workspace proxies.",
+			Flag:        "proxy-health-interval",
+			Env:         "CODER_PROXY_HEALTH_INTERVAL",
+			Default:     (time.Minute).String(),
```

Comment:
```suggestion
			Default:     "1m",
```

Reply:
https://github.com/coder/coder/blob/5ff22ac8e85fdbb2a8f00024d090eba337d267a7/codersdk/deployment.go#L535-L535

https://github.com/coder/coder/blob/5ff22ac8e85fdbb2a8f00024d090eba337d267a7/codersdk/deployment.go#L1087-L1087

We do this pattern for all other durations :shrug:

Reply:
Ah never mind then